todoist_api_python/models.py

```diff
@@ -358,3 +359,151 @@ def from_dict(cls, obj):
             access_token=obj["access_token"],
             state=obj["state"],
         )
+
+
+@dataclass
+class Item:
+    id: str
+    """The ID of the task."""
```

Comment:
Could you remove all the docstrings from this point and below, so the code looks the same as the rest of the file?

These dataclasses are mainly for input validation, and the descriptions of the properties are in the [Todoist API doc](https://developer.todoist.com) so we don't need to duplicate them here.